README.md

```diff
@@ -0,0 +1,175 @@
+# Mongoose Event Logger
+
+> Granular change logging for your [Mongoose](http://mongoosejs.com/) models
+
+> Which model was updated, what was changed, when and by who?
+
+
+[![NPM version](https://img.shields.io/npm/v/mongoose-event-logger.svg)](https://npmjs.org/package/mongoose-event-logger)
+[![Build status](https://img.shields.io/travis/teamintrepid/mongoose-event-logger.svg)](https://travis-ci.org/teamintrepid/mongoose-event-logger)
+
+
+## Installation
+
+```sh
+npm install mongoose-event-logger
+```
+
+## Quick Start
+
+#### Setup
```

Comment:
Somewhere we should mention that the logger opens the database connection and the connection should be closed before the app can exit cleanly by awaiting for `logger.stop()`. It is also important because if the process if prematurely killed, the events that are currently in the pipeline may be lost.
There is a configuration option for the Logger `attachSignalListeners` (which isn't set by default) that allows it to hook on the process termination signals and do the cleanup by itself.